00-RELEASENOTES

```diff
@@ -1,16 +1,213 @@
-Hello! This file is just a placeholder, since this is the "unstable" branch
-of Valkey, the place where all the development happens.
+Valkey 9.0 release notes
+========================
 
-There is no release notes for this branch, it gets forked into another branch
-every time there is a partial feature freeze in order to eventually create
-a new stable release.
+Upgrade urgency levels:
 
-Usually "unstable" is stable enough for you to use it in development environments
-however you should never use it in production environments. It is possible
-to download the latest stable release here:
+| Level    | Meaning                                                             |
+|----------|---------------------------------------------------------------------|
+| LOW      | No need to upgrade unless there are new features you want to use.   |
+| MODERATE | Program an upgrade of the server, but it's not urgent.              |
+| HIGH     | There is a critical bug that may affect a subset of users. Upgrade! |
+| CRITICAL | There is a critical bug affecting MOST USERS. Upgrade ASAP.         |
+| SECURITY | There are security fixes in the release.                            |
 
-    https://valkey.io/download/
+Valkey 9.0.0-rc1 - August 12, 2025
+----------------------------------
 
-More information is available at https://valkey.io
+Upgrade urgency LOW: This is the first release candidate of Valkey 9.0.0, with
+performance improvements, atomic slot migrations, hash field expiration, and numbered databases in cluster mode,
+as well as a whole host of new items as listed below.
 
-Happy hacking!
+### New Features
+* Introduce atomic slot migration by @murphyjacob4 (#1949)
+* Introduce HASH items expiration by @ranshid (#2089)
+* support negative filtering for client command filters by @soloestoy (#2378)
+* Add support for automatic client authentication via TLS certificate fields by @omanges (#1920)
+* Add SAFE option to SHUTDOWN to reject shutdown in unsafe situations by @enjoy-binbin (#2195)
+* Support BYPOLYGON option for GEOSEARCH by @KarthikSubbarao (#1809)
+* Add DELIFEQ command by @LinusU (#1975)
+* Add multi-database support to cluster mode by @xbasel (#1671)
+* Introduce MPTCP for replica by @pizhenwei (#1961)
+* Add --hotkeys-count option for valkey-cli by @hwware (#1933)
+* Add sentinel_total_tilt to sentinel INFO sentinel by @carlosfu (#1904)
+* Introduce MPTCP by @pizhenwei (#1811)
+* Allow dynamic modification of io-threads num by @ayush933 (#2033)
+* Client Commands Extended Filtering by @sarthakaggarwal97 (#1466)
+
+### New Configs
+* Auto-failover on shutdown unified config by @zuiderkwast (#2292)
```

Comment:
Should we combined related PRs in some way?

This one about the configs is just a change to the not yet released feature "Trigger manual failover on SIGTERM / shutdown to cluster primary by @enjoy-binbin (#1091)" which is listed under Cluster.

It might be easier to not combine them. Then the readers will have to follow the links in each PR to get the full picture.

Reply:
Yeah, this was something I originally commented about, but it seemed like too much work.

Reply:
this section looks new (and incomplete?). I am also inclined to not having a top level "new config" section.